tests/test_singleton.py

```diff
@@ -53,6 +58,26 @@ def test_dbos_singleton(cleanup_test_databases: None) -> None:
     wfhr: WorkflowHandle[str] = DBOS.retrieve_workflow(wh.get_workflow_id())
     assert wfhr.workflow_id == wh.get_workflow_id()
 
+    # Test wrapper class methods
+    wrapper_inst = DBOSTestWrapperMethods("dynamicconfig")
+    res = wrapper_inst.test_workflow("x", "y")
+    assert res == "y1x"
+
+    wh = DBOS.start_workflow(wrapper_inst.test_workflow, "x", "y")
+    assert wh.get_result() == "y1x"
+    stati = DBOS.get_workflow_status(wh.get_workflow_id())
+    assert stati
+    assert stati.config_name is None
+    assert stati.class_name is None
```

Comment:
Why are these both None? How is this recoverable?

Reply:
Both are None, but the workflow can get the `self` from its closure. I can add a test for recovery.